### DISCUSSION TITLE
Hotkeys

### DISCUSSION THREAD

**Discussion by nikmtl** [JakobJFL/Hue-Browser-Controller#22] [Ideas]
Hey, i would love to be able to control my lights via hotkeys in the desktop app. So that it is running in the background and you can turn light on and off with just pressing a combination on your keyboard.

U have done a great job though.

**Comment by JakobJFL** [comment]
Hey, that's a good idea. I actually have that on my computer but it's something I made myself and I can easily make it so outer people can use it. Currently I'm working on another project so I don't have that much time. But I will se if I get time to make something people can use.
